util/passport.js

```diff
@@ -31,6 +36,73 @@ module.exports = function(passport) {
         });
     }));
 
+    passport.use(new GoogleStrategy({
+        clientID: config.google.clientID,
+        clientSecret: config.google.clientSecret,
+        callbackURL: config.baseURL + "/auth/google/callback"
+    }, function(req, accessToken, refreshToken, profile, done) {
+        function renderResponse(errorMsg) {
+            return responseFactory.sendRenderedResponse("public/signup", req, req.res, { captcha: app.recaptcha.render(), error: { message: errorMsg || "An unknown error occurred" }, username: "" });
+        }
+        User.findOne({ name: "google_" + profile.id }, function(err, user) {
```

Comment:
Eh, there's not really any reason to. It just kind of defeats the purpose of it. Also, we don't really use the username for anything important... If you end up showing pixel blame or something, you could always have it be a "Screen name" which can be changed in settings that defaults to the user's username (or username on the OAuth service if it's OAuth)